### PR TITLE
update-regex.icu_fallback-encoding

### PR DESCRIPTION
**Pull request by lukewrites**
The `string.regex.icu_setup` test file is encoded in something other than utf-8, which causes Python to break when the test is run on Mac. This PR updates the test and expected. It should allow enabling this test on some data sources where it is currently disabled.

Testing:
- StandardDB: hyper and hyper_native pass on Mac and Linux.

### CODE REVIEW

File: tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regex.icu_fallback.2.txt [hunk 1]
```diff
@@ -0,0 +1,909 @@
+<results>
+  <test name='REGEXP_EXTRACT(&apos;http://academic.research.microsoft.com/Author/232694&apos;, &apos;((https|http)+:\/\/+(www\.)?+[a-z0-9\.:-]+\/)&apos;)'>
+    <sql>
+      SELECT TABLEAU.REGEXP_EXTRACT(&apos;http://academic.research.microsoft.com/Author/232694&apos;, E&apos;((https|http)+:\\/\\/+(www\\.)?+[a-z0-9\\.:-]+\\/)&apos;) AS &quot;TEMP(Test)(368590675)(0)&quot;&#10;FROM &quot;Extract&quot;.&quot;Extract&quot; &quot;Extract&quot;&#10;HAVING (COUNT(1) &gt; 0)&#10;    </sql>
+    <query-time>
+      5.000&#10;    </query-time>
```
Comment (rosswbrown) [inline tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regex.icu_fallback.2.txt:6]: sql and query-time elements are suppressed in the other expected files

File: tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.icu_fallback.txt [hunk 2]
```diff
@@ -23,8 +23,8 @@ REGEXP_EXTRACT_NTH([str1],'(\w+)\s{1}+(\w+)\Z',1)
 REGEXP_MATCH('0123456789', '(?=\d{10})\d{5}')
 REGEXP_REPLACE('0123456789', '(?=\d{10})\d{5}','NewString') 
 REGEXP_MATCH('iamastring', '(?!test)iam\w+')
-REGEXP_MATCH('B���T�F�L100','(?<=B���T�F�L)\d{3}')
-REGEXP_REPLACE('B���T�F�L100B���T�F�L','(?<=B���T�F�L)\d{3}','Replaced')
+REGEXP_MATCH('BÊÂÙTÌFÜL100','(?<=BÊÂÙTÌFÜL)\d{3}')
```
Comment (rosswbrown) [inline tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.icu_fallback.txt:26]: Comparing the different expected files. Are you sure we shouldn't just update them both after this change?  It seems like this is going to change the inputs in all cases now.

File: tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regex.icu_fallback.2.txt (hunk 1 above)
Comment (lukewrites) [reply tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regex.icu_fallback.2.txt:6]: TDVT ignores those sections when doing the comparison, but I will make sure they are removed to keep things tidy.

File: tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.icu_fallback.txt (hunk 2 above)
Comment (lukewrites) [reply tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.icu_fallback.txt:26]: Agreed, I'll update after getting the results of a composite StandardDBs run.

File: tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regex.icu_fallback.2.txt (hunk 1 above)
Comment (lukewrites) [reply tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regex.icu_fallback.2.txt]: Removed in 783f54290694aec4f079b74d08009d1e79203ab9

File: tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.icu_fallback.txt (hunk 2 above)
Comment (lukewrites) [reply tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.icu_fallback.txt:26]: I removed all other expecteds (e436c01f0c3caeb5a44976822f62a025e85855f1), added `hyper` and `hyper_native` to the standard suite for Mac & Linux, and run StandardDBs - everything passes.

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**